e2e/tests/cohorts/cohorts-review.spec.ts

```diff
@@ -119,11 +120,38 @@ describe('Cohort review tests', () => {
     // select a review status
     const participantStatus2 = await sidebarContent.selectReviewStatus(ReviewStatus.Included);
 
-   // verify if the same Annotations Name also displays for the next Participant ID.
+    // verify if the same Annotations Name also displays for the next Participant ID.
     const annotationTextBoxName = await sidebarContent.getAnnotationsName();
     expect(annotationTextBoxName).toEqual(newAnnotationName);
 
+    // click on the annotations EDIT button
+    await sidebarContent.getAnnotationsEdit().then(btn => btn.click());
+
+     // create new annotation name
+     const newAnnotationRename = makeRandomName();
+
+    // the edit-delete annotations modal displays
+    const editDeleteAnnotationsFieldModal = new EditDeleteAnnotationsModal(page); 
+
+    // edit the annotation field name
+    await editDeleteAnnotationsFieldModal.clickRenameAnnotationsName(newAnnotationRename);
+
+    // verify that the Annotation textbox field is displaying the new name
+    const annotationNewTextBoxName = await sidebarContent.getAnnotationsName();
+    expect(annotationNewTextBoxName).toEqual(newAnnotationRename);
+
+    await participantDetailPage.goToThePriorParticipant();
+    await participantDetailPage.waitForLoad();
+
+    // verify that the prior participant is displaying the same annotation field name
+    expect(annotationNewTextBoxName).toEqual(newAnnotationRename);
+
+    // click the annotations edit  button to delete the annotation textbox field
+    await sidebarContent.getAnnotationsEdit().then(btn => btn.click());
+    await editDeleteAnnotationsFieldModal.deleteAnnotationsName();
```

Comment:
do you think delete textbox needs a verification?

Reply:
Agree! we need to have a verification step for delete textbox. Will add it!

Reply:
@aweng98 I tried adding a verification step for the textbox deletion but did not succeed.